### PR TITLE
fix informers for operator namespace

### PR DESCRIPTION
**Pull request by adambkaplan**

### CODE REVIEW

File: pkg/operator/observe_config.go [hunk 1]
```diff
@@ -75,18 +75,18 @@ func NewConfigObserver(
...
 
 	c.operatorConfigSynced = operatorConfigInformer.Informer().HasSynced
-	c.configmapSynced = kubeInformersForOpenshiftCoreOperators.Core().V1().ConfigMaps().Informer().HasSynced
+	c.configmapSynced = kubeInformersForOperator.Core().V1().ConfigMaps().Informer().HasSynced
 	c.imageConfigSynced = configInformer.Config().V1().Images().Informer().HasSynced
 	c.buildConfigSynced = configInformer.Config().V1().Builds().Informer().HasSynced
 
 	operatorConfigInformer.Informer().AddEventHandler(c.eventHandler())
-	kubeInformersForOpenshiftCoreOperators.Core().V1().Namespaces().Informer().AddEventHandler(c.eventHandler())
+	kubeInformersForOperator.Core().V1().ConfigMaps().Informer().AddEventHandler(c.eventHandler())
```
Comment (adambkaplan) [inline pkg/operator/observe_config.go]: Watch the config maps for changes to `openshift-controller-manager-images`

Comment (bparees) [reply pkg/operator/observe_config.go]: @sanchezl you might also have input here.

**Comment by bparees** [comment]
@adambkaplan please rebase and lgtm.

**Pull request merged**